ENGAGEHF/Medications/RowLabel/RecommendationSymbolColor.swift

```diff
@@ -0,0 +1,36 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+enum RecommendationSymbolColor {
+    static func color(for type: MedicationRecommendationType) -> Color {
```

Comment:
Why use static functions here in this type instead of computed properties right on the MediationRecommendationType?